app/routes/comment.routes.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (app) => {
+  const comment = require("../controllers/comment.controller.js");
+  const { authenticate } = require("../authorization/authorization.js");
+  var router = require("express").Router();
+
+  router.get(
+    "/resume/review/:reviewId/comment",
+    [authenticate],
+    comment.getAll
+  );
+
+  router.post(
+    "/resume/review/:reviewId/comment",
+    [authenticate],
+    comment.create
+  );
+
+  router.put("/resume/review/comment/:id", [authenticate], comment.update);
+
+  router.delete("/resume/review/comment/:id", [authenticate], comment.destroy);
```

Comment:
Almost all of the test cases said "given an admin", but none of these routes are admin secured

Reply:
Fixed